convene-web/config/routes.rb

```diff
@@ -30,7 +32,6 @@
   resources :guides, only: %i[index show]
 
   constraints BrandedDomain.new(Space) do
-    root 'spaces#show'
```

Comment:
This would break `https://meet.zinc.coop` or other spaces with custom domains. We _probably_ have a test for that? Maybe? Or if we don't, we may want to add one.

That said, there is probably some benefit in making the `root 'spaces#show'` the global root; and have it render a `ConveneSpace` that we expect to always exists (kinda like how `SystemTestSpace` and `DemoSpace` exist in development and review apps). As we build out furniture, it may even serve as the formal public landing page of https://convene.zinc.coop!

Reply:
I brought back `space#show` as the root route everywhere, and added a dummy blank `Space` to show when there is no space specified.